drivers/serial/uart_mcux_lpuart.c

```diff
@@ -83,8 +138,8 @@ static void mcux_lpuart_poll_out(const struct device *dev, unsigned char c)
 	struct mcux_lpuart_data *data = dev->data;
 #endif
 
-	while (!(LPUART_GetStatusFlags(config->base)
-		& kLPUART_TxDataRegEmptyFlag)) {
+	while (!(LPUART_GetStatusFlags(config->base) &
+		 kLPUART_TxDataRegEmptyFlag)) {
```

Comment:
In the future, you should resist the desire to do whitespace changes in the file you are adding changes to. It creates noise to the reviewers.

Reply:
This was a remnant from the uncrustify auto-formatting (I realize this has been replaced by clang-tidy, but the documentation I read when starting to work on this indicated to format all changes with the uncrustify config that was checked into the repo).
Will keep this in mind for the future, however!